src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -71,6 +82,24 @@ func newBootstrapCmd() *cobra.Command {
 	return bootstrapCmd
 }
 
+func getConfigFromYaml(filePath string) (apiv1.BootstrapConfig, error) {
+	config := apiv1.BootstrapConfig{}
+	config.SetDefaults()
```

Comment:
config.SetDefaults() should be _after_ reading the config file, no?

Reply:
Right now, if set after the yaml config values are set the `SetDefaults()` method would overwrite our config since it doesn't check for existing values.